### PR TITLE
docs: block controls visible #815

### PR DESCRIPTION
**Pull request by Bishwas-py**
The code block controls were missing in this block `documentation/docs/98-reference/54-types.md`, as mentioned in https://github.com/sveltejs/svelte.dev/issues/815.

This PR assign `link: true`, which eventually causes the controls and headers to be visible.

image
---

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### Edits

- [x] Please ensure that 'Allow edits from maintainers' is checked. PRs without this option may be closed.

### CODE REVIEW

**Comment by Rich-Harris** [comment]
preview: https://svelte-dev-git-preview-kit-12969-svelte.vercel.app/

_this is an automated message_

**Comment by Bishwas-py** [comment]
Permalink to the block: https://svelte-dev-git-preview-kit-12969-svelte.vercel.app/docs/kit/types#:~:text=.svelte%2Dkit/types/src/routes/%5Bfoo%5D/%5Bbar%5D/%5Bbaz%5D/%24types.d

File: documentation/docs/98-reference/54-types.md [hunk 1]
```diff
@@ -154,4 +154,73 @@ The `app.d.ts` file is home to the ambient types of your apps, i.e. types that a
 
 Always part of this file is the `App` namespace. This namespace contains several types that influence the shape of certain SvelteKit features you interact with.
 
-> TYPES: App
+## Error
```
Comment (dummdidumm) [inline documentation/docs/98-reference/54-types.md:157]: This looks like an accidental change that should be reverted

Comment (Bishwas-py) [reply documentation/docs/98-reference/54-types.md]: Changes reverted!

**Review by eltigerchino — APPROVED** [review]

**Comment by eltigerchino** [comment]
Thanks!

**Pull request merged**